pkg/apis/pipeline/v1beta1/task_validation.go

```diff
@@ -319,10 +319,6 @@ func (p ParamSpec) ValidateType() *apis.FieldError {
 // definition of `properties` section and the type of a PropertySpec is allowed.
 // (Currently, only string is allowed)
 func (p ParamSpec) ValidateObjectType() *apis.FieldError {
-	if p.Type == ParamTypeObject && p.Properties == nil {
-		return apis.ErrMissingField(fmt.Sprintf("%s.properties", p.Name))
-	}
-
```

Comment:
That's right. We cannot have this check because there might be no `properties` field defined when users are propagating parameters. Good catch though. When we explicitly declare the task without parameter propagation we want this check. I will fix accordingly and ping you later to verify.